src/components/Account/Create.tsx

```diff
@@ -52,40 +62,99 @@ export const AccountCreate = () => {
       id='process-create-form'
       direction='column'
       gap={6}
-      mt={6}
       onSubmit={(e) => {
         e.stopPropagation()
         e.preventDefault()
         handleSubmit(onSubmit)(e)
       }}
     >
-      <FormControl isInvalid={!!errors.name}>
-        <Input
-          type='text'
-          {...register('name', { required })}
-          mb={1}
-          placeholder={t('form.account_create.title_placeholder').toString()}
+      <Text fontWeight='light'>
+        <Trans
+          i18nKey='new_organization.description1'
+          components={{
+            span: <Text as='span' fontWeight='bold' />,
+          }}
         />
-        {!!errors.name ? (
-          <FormErrorMessage>{errors.name?.message?.toString()}</FormErrorMessage>
-        ) : (
+      </Text>
+      <Text fontWeight='light'>
+        <Trans
+          i18nKey='new_organization.description2'
+          components={{
+            span: <Text as='span' fontWeight='bold' />,
+          }}
+        />
+      </Text>
+      <Box px={{ base: 5, md: 10 }} pt={5} pb={10}>
+        <FormControl isInvalid={!!errors.name} mb={5}>
+          <FormLabel fontWeight='bold'>*{t('new_organization.name')}</FormLabel>
+          <Input
+            type='text'
+            {...register('name', { required })}
+            mb={1}
+            placeholder={t('form.account_create.title_placeholder').toString()}
+          />
+          {!!errors.name && <FormErrorMessage>{errors.name?.message?.toString()}</FormErrorMessage>}
+        </FormControl>
+
+        <FormControl>
+          <Textarea
+            {...register('description')}
+            placeholder={t('form.account_create.description_placeholder').toString()}
+          />
           <FormHelperText>
             <InfoOutlineIcon />
-            <Text>{t('form.account_create.title_helper')}</Text>
+            <Text> {t('form.account_create.description_helper')}</Text>
```

Comment:
Why add an empty space before the translation? Give it a margin, padding, or add the space in the translation itself (think how should be that space if the reading was RTL).